mappings/net/minecraft/recipe/impl/ShieldRecipes.mapping

```diff
@@ -1,8 +1,8 @@
-CLASS none/agz
-	CLASS none/agz$a
+CLASS none/agz net/minecraft/recipe/impl/ShieldRecipes
```

Comment:
I would have probably called it RecipeShield and RecipeShield.Design, respectively.